torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -13,9 +13,20 @@
 namespace torch {
 namespace jit {
 
-static bool texpr_fuser_enabled = true;
+static bool texpr_fuser_enabled_ = false;
```

Comment:
Nit: why underscore at the end?

Reply:
previous code used `texpr_fuser_enabled` directly, whereas code should now use the function `tensorExprFuserEnabled`.  the underscore is my attempt at "privatizing" the variable so folks don't use it directly